subsys/logging/Kconfig

```
@@ -279,6 +279,40 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FLASH
+	bool "Enable FLASH backend"
+	depends on FLASH
```

```suggestion
	depends on FILE_SYSTEM
```

This patch shouldn't depends on FS implementation but on VFS API avielable.